standards_lab/api/views.py

```diff
@@ -27,8 +28,10 @@ def OK(project):
     return JsonResponse(project)
 
 
-def FAILED(error):
-    return JsonResponse({"status": "FAILED", "error": error})
+def FAILED(error, status_code=500):
+    r = JsonResponse({"status": "FAILED", "error": error})
```

Comment:
I like that we now have the right http error code. Just to double check -This shouldn't effect other calls/handlers on the project page should it resolve even if you get 500 ?

Reply:
Pretty sure yes, this is in fact the essence of #134 - as you can see in the example there the load file handler still fires even if it's a non-200!